packages/presentation/src/components/PDFViewer.svelte

```diff
@@ -67,7 +66,7 @@
   .pdfviewer-container {
     display: flex;
     flex-direction: column;
-    width: 45rem;
+    width: 55rem;
```

Comment:
Why 55? I think we need min-width and max-width

Reply:
Probable.